test/requirements-test.txt

```diff
@@ -16,6 +16,8 @@ git+https://github.com/microsoft/TransformerCompression.git ; python_version >=
 mlflow>=2.4.0
 neural-compressor
 nncf==2.7.0
+nvidia-modelopt~=0.11.0
+onnx-graphsurgeon
```

Comment:
```suggestion
nvidia-modelopt[onnx]~=0.11.0
onnx-graphsurgeon
```

Reply:
This change cannot be made due to dependency clash between the ci's ort package requirements and the extras requirements. Please refer to the discussion above for more details https://github.com/microsoft/Olive/pull/1135#discussion_r1592736575